Fix typo added in lp1668771-resolved-switch-cache-option-to-a-tri-state-option-s.patch

--- systemd-237.orig/src/shared/resolve-util.h
+++ systemd-237/src/shared/resolve-util.h
@@ -65,10 +65,6 @@ int config_parse_resolve_support(const c
 int config_parse_dnssec_mode(const char *unit, const char *filename, unsigned line, const char *section, unsigned section_line, const char *lvalue, int ltype, const char *rvalue, void *data, void *userdata);
 int config_parse_dns_cache_mode(const char *unit, const char *filename, unsigned line, const char *section, unsigned section_line, const char *lvalue, int ltype, const char *rvalue, void *data, void *userdata);
 
-CONFIG_PARSER_PROTOTYPE(config_parse_resolve_support);
-CONFIG_PARSER_PROTOTYPE(config_parse_dnssec_mode);
-CONFIG_PARSER_PROTOTYPE(config_parse_dns_cache_mode);
-
 const char* resolve_support_to_string(ResolveSupport p) _const_;
 ResolveSupport resolve_support_from_string(const char *s) _pure_;
 
